test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -416,6 +449,8 @@ export default class PerfCompare extends Component {
             curVariantData["baselineProduct"] = this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].product;
             curVariantData["testProduct"] = this.state.benchmarkRuns.benchmarkRunTest.parsedVariants[curMatchingTestVariantIndex].product;
             curVariantData["summary"] = "";
+            curVariantData["baselineMachine"] = this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i].machine;
+            curVariantData["testMachine"] = this.state.benchmarkRuns.benchmarkRunTest.parsedVariants[i].machine;
```

Comment:
Can we make `this.state.benchmarkRuns.benchmarkRunBaseline.parsedVariants[i]` a variable? It is used in 14 different places.

Reply:
Updated!